### PR TITLE
Replace infographic with Fairy's internship project 2015 (Fall)

### PR DESCRIPTION
**Pull request by jeepurs**
Deployment 'How-to' infographic has been updated by Fairy Bui with results from her interviews with past deployments.

### CODE REVIEW

**Comment by jeepurs** [comment]
@aronasorman Will you review?

**Comment by aronasorman** [comment]
Wahoo! Will review tomorrow morning.

File: fle_site/apps/ka_lite/templates/ka_lite/infographic.html [hunk 1]
```diff
@@ -7,19 +7,5 @@
...
-	        if (d.getElementById(id)) { return;}
-	        js=d.createElement("script"); js.id=id; js.async=true;
-	        js.src="https://magic.piktochart.com/assets/embedding/embed.js";
-	        ref.parentNode.insertBefore(js, ref);
-	    }(document));
-	</script>
+	<div class="piktowrapper-embed" pikto-uid="9810689-implementations-infographic" style="height: 300px; position: relative;"><div class="embed-loading-overlay" style="width: 100%; height: 100%; position: absolute; text-align: center;"><img width="60px" alt="Loading..." style="margin-top: 100px" src="https://magic.piktochart.com/loading.gif"/><p style="margin: 0; padding: 0; font-family: Lato, Helvetica, Arial, sans-serif; font-weight: 600; font-size: 16px">Loading...</p></div><div class="pikto-canvas-wrap"><div class="pikto-canvas"></div></div></div><script>(function(d){var js, id="pikto-embed-js", ref=d.getElementsByTagName("script")[0];if (d.getElementById(id)) { return;}js=d.createElement("script"); js.id=id; js.async=true;js.src="https://magic.piktochart.com/assets/embedding/embed.js";ref.parentNode.insertBefore(js, ref);}(document));</script>
```
Comment (aronasorman) [inline fle_site/apps/ka_lite/templates/ka_lite/infographic.html]: This long `div` is unreadable. You also placed the style inline. Any reason why?

**Comment by jeepurs** [comment]
I copy/pasted the html output provided to me by Piktochart.  I can stylize it if you give me some guidance @aronasorman

**Comment by aronasorman** [comment]
Will clean up with @jeepurs tomorrow.

File: fle_site/apps/ka_lite/templates/ka_lite/infographic.html [hunk 2]
```diff
@@ -7,19 +7,5 @@
...
-</div>
-	<script>
-	    (function(d){
-	        var js, id="pikto-embed-js", ref=d.getElementsByTagName("script")[0];
-	        if (d.getElementById(id)) { return;}
-	        js=d.createElement("script"); js.id=id; js.async=true;
-	        js.src="https://magic.piktochart.com/assets/embedding/embed.js";
-	        ref.parentNode.insertBefore(js, ref);
-	    }(document));
-	</script>
```
Comment (aronasorman) [inline fle_site/apps/ka_lite/templates/ka_lite/infographic.html]: Please add comment saying, this is autogenerated by piktochart, it fetches from their site. And that we shouldn't mess with it. Excuse the long line in the tags.

**Pull request merged**